conformance/service_import.go

```diff
@@ -139,13 +144,26 @@ var _ = Describe("", func() {
 				reportNonConformant(fmt.Sprintf("The value %q is not a valid IP", serviceImport.Spec.IPs[0])))
 		})
 
+	Specify("Annotations and labels should be empty if not explicitly exported",
```

Comment:
Test cases require creating a namespace and other resources so they aren't cheap. As such, I don't think we need a specific test case for this. This would naturally be verified as part of the spec on L258 below.

Reply:
Makes sense! I removed it